person.rb

```diff
@@ -0,0 +1,22 @@
+# person.rb
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age: 18, name: 'Unknown', parent_permission: true)
+    @id = rand(1...1000) # Generate a random ID within the range 1-999
```

Comment:
- [ ] Good job adding the Person class, however, as the previous reviewer said, the parameter `age` should not be an optional parameter, it should be required. Make sure you removed `18`. we have something like this:
````ruby
def initialize(age, name: "Unknown", parent_permission: true)
    @id = Random.rand(1..1000)
````